common/build.gradle

```diff
@@ -13,12 +17,13 @@ dependencies {
     testImplementation 'androidx.test:core:1.2.0'
     testImplementation 'androidx.test:runner:1.2.0'
     testImplementation 'androidx.test:rules:1.2.0'
-    testImplementation 'androidx.test.ext:junit:1.1.1'
+    testImplementation "org.junit.jupiter:junit-jupiter-api:$junit5Version"
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit5Version"
+    testImplementation 'org.json:json:20210307'
```

Comment:
This is adding the latest official json library to the tests of the `common` module. As mentioned We have to either do this, or mock all json related classes we were using. In both cases, we would lose the ability to test with the real implementation of those classes (the one included on the android sdk)